### PR TITLE
Eliminate lang function

### LINKED ISSUE #767: Delete all `lang` function and use I18n interface instead
# Details

Thanks to Ruin0x11's much efforts, almost all of hard-coded `lang()` function have been deleted in the code base. Now that it is possible to delete all the rest by using I18N interface.

### PR DESCRIPTION
**Pull request by ki-foobar**
# Related Issues

Close #767.

# Summary

Externalizes all `lang` functions to `runtime/locale/*/*.hcl`s.

# Tasks

Check these text both in English and Japanese.

- [x] Message when some buff is applied/removed.
- [x] Message when some Rapid Growth buff is applied.
- [x] Message when your karma change.
  - [x] No longer a criminal       (EDIT: add missing period in English)
  - [x] Become a criminal
- [x] Message when you gain a new body part. (for mutant)
- [x] Guild' names (including None)
- [x] Something falls from your backpack.
- [x] The counter of an atomic bomb.
- [x] Map effects:
  - [x] Acid pool
  - [x] Fire pillar
  - [x] Spilt potion
  - [x] Cobweb
  - [x] Mist of illusion
- [x] "It's mine."/"Too heavy!"/"No way."(furniture)/"I don't want it."(garbage)/"Never!"(cargo goods)
- [x] Last words(lastwords.txt/lastwords-e.txt)
- [x] Message when you assign a shortcut key.
- [x] Norne's guide
- [x] Scenario
- [x] Politics posts
- [x] Items are stacked.
- [x] Message when some item has been damaged by acid(acidproof or not).
- [x] Some items are destroyed because too many items exist in the current map.
- [x] Item name
  - [x] Parentheses of miracle(<>) and godly({})
  - [x] (Charges: N)
  - [x] (Scary)/(Dreadful)
- [x] Help
- and so on...

### CODE REVIEW

File: runtime/locale/en/ctrl_inventory.hcl [hunk 1]
```diff
@@ -202,12 +202,7 @@ locale {
             }
 
             take_ally {
-                refuse_dialog {
-                    _0 = "\"No!\""
-                    _1 = "\"It's mine.\""
-                    _2 = "\"Get off!\""
-                    _3 = "\"Never.\""
-                }
+                refuse_dialog = "\"It's mine.\""
```
Comment (Ruin0x11) [inline runtime/locale/en/ctrl_inventory.hcl]: Is this the correct translation? Before there used to be four items.

File: runtime/locale/en/food.hcl [hunk 2]
```diff
@@ -248,8 +248,6 @@ locale {
                 deteriorates = "${name(_1)}${his_owned(_1)} ${_2} deteriorates."
             }
 
-            growth = "${name(_1)}${his_owned(_1)} ${_2} enters a period of rapid growth."
```
Comment (Ruin0x11) [inline runtime/locale/en/food.hcl]: Was this removed unintentionally?

File: runtime/locale/en/ctrl_inventory.hcl (hunk 1 above)
Comment (ki-foobar) [reply runtime/locale/en/ctrl_inventory.hcl]: 2nd-4th items were defined, but unused.

File: runtime/locale/en/food.hcl (hunk 2 above)
Comment (ki-foobar) [reply runtime/locale/en/food.hcl]: Growth buff's message was moved to `buff.hcl`.

**Comment by ki-foobar** [comment]
## Fire

- Blanket protects one's item.
  - [ ] JP
  - [ ] EN (you)
  - [ ] EN (others)
- Blanket turns to dust.
  - [ ] JP
  - [ ] EN
- Item someone equip(s) turns to dust.
  - [ ] JP
  - [ ] EN (you)
  - [ ] EN (others)
- One's item(s) turn(s) to dust.
  - [ ] JP
  - [ ] EN (one item, you)
  - [ ] EN (one item, others)
  - [ ] EN (several items, you)
  - [ ] EN (several items, others)
- Item(s) on the ground turn(s) to dust.
  - [ ] JP
  - [ ] EN (one item)
  - [ ] EN (several items)

## Cold

- Blanket protects one's item.
  - [ ] JP
  - [ ] EN (you)
  - [ ] EN (others)
- Blanket is broken to pieces.
  - [ ] JP
  - [ ] EN
- One's item(s) break(s) to pieces.
  - [ ] JP
  - [ ] EN (one item, you)
  - [ ] EN (one item, others)
  - [ ] EN (several items, you)
  - [ ] EN (several items, others)
- Item(s) on the ground break(s) to pieces.
  - [ ] JP
  - [ ] EN (one item)
  - [ ] EN (several items)

**Comment by ki-foobar** [comment]
Test these things.

# Locations

> You see {location}. (5 locations)

- [ ] JP
- [ ] EN

# Items

## Messages

> You can only use cargo items on the surface.

- [ ] JP
- [ ] EN

## Armor class

> Heavy/medium/light

- [ ] JP
- [ ] EN

## Gift rank (rank 1-6)

- [ ] JP
- [ ] EN

## Kitty bank rank (500/2k/10k/50k/500k/5M/500M)

- [ ] JP
- [ ] EN

## Bait rank (6 kinds)

- [ ] JP
- [ ] EN

**Comment by ki-foobar** [comment]
`lang()` calls related to item name still exist. See also #840.

**Pull request merged**